zulip_bots/zulip_bots/bots/susi/test_susi.py

```diff
@@ -39,7 +39,7 @@ def test_help(self) -> None:
 
     def test_issue(self) -> None:
         request = 'hi'
-        bot_response = 'Hello!'
+        bot_response = 'Hello!\n'
```

Comment:
will this cover all these changes ?

Reply:
@tonythomas01 yes, I think this will be sufficient. The bot can reply with different answers as the answers come from the SUSI server. So this test case will just check if the bot APIs are working properly.

Reply:
perfect.